index.rst

```diff
@@ -6,7 +6,158 @@ Roadmap for Community Alert Filters with the ANTARES Broker
+
+* prepare and advertise call input from the community
+* review the input and draft ~5 initial community filters
+
+FY25 Q3, Apr-Jun 2025 (or when LSST alerts begin):
+
+* implement ~5 initial community filters
+* release documentation and use-case tutorials
+
+FY25 Q4, Jul-Sep 2025:
+
+* prepare validation report and ‘lessons learned’ for the community
+
+FY26 Q1, Oct-Dec 2026:
+
+* second round of community input for additional community filters
+
+
+Technical constraints
+=====================
+
+The following can be considered as rough guidelines.
+
+Alert contents
+--------------
+
+See Section 3.5 of the Rubin Data Products Definitions Document (`LSE-163 <https://lse-163.lsst.io/>`_) for a list of alert packet contents, and `sdm-schemas.lsst.io/apdb.html <https://sdm-schemas.lsst.io/apdb.html>`_ has the as-implemented column names.
+
+The community filters will use only the contents of the LSST alert packet plus any additional or derived data added by ANTARES, such as cross-matches with external (non-Rubin) catalogs performed by ANTARES.
+
+The contents of the alert packets might evolve over Rubin Operations, thereby enabling new community filters.
+For example, the final set of time series features to be computed during Alert Production remains to be determined (`DMTN-118 <https://dmtn-118.lsst.io/>`_).
+
+Once a Rubin data release exists (DP2, DR1), the community filters will be able to query and use proprietary LSST data, such as multi-year light curves or the photo-z of potential host galaxies.
+
+Processing limits
+-----------------
+
+The computational resources will be shared across all community filters, and thus what is available for a given filter will depend on the number of filters, their throughput, and the actual number of alerts per visit.
+
+The LSST observing strategy will make one visit every ~30 seconds.
+Any filter which takes, on average during the night, longer than 30 seconds per visit (~10,000 alerts) to process alerts from one visit will fall behind.
+
+* Maximum latency per visit: 30 seconds (average over the night)
+
+Follow-up feasibility
+---------------------
+
+The LSST Alerts: Key Numbers document (`DMTN-102 <https://dmtn-102.lsst.io/>`_) document predicts an average of 10,000 alert-triggering astrophysical sources per visit, with a maximum of >30,000 in Galactic fields.
+The community filters should significantly sub-select the number of alerts per visit.
+Filters that pass too many alerts do not sufficiently inform follow-up prioritization, which is the point of the filter.
+
+* Average number of passing alerts per visit: 20 alerts
+* Maximum number of passing alerts per visit: 100 alerts
+
+Machine-learned models
+----------------------
+
+Community filters can be based on a trained ML model.
+
+However, any community filters based on machine learning models would need to be trained and validated by volunteer members of the science community, training is beyond the scope of the Rubin CST.
```

Comment:
maybe add a sentence about making sure models are technically feasible in ANTARES?  I don't know what packages you can import, for instance.

Reply:
From a discussion with Tom, ANTARES can do ML models (but not active learning models), but that packages required for filters need to be vetted. Will add more detail on the package vetting.

Reply:
And yes, use of trained ML models is technically feasible for ANTARES (they're already doing it)

Reply:
```suggestion
The use of third-party software packages in community filters will be evaluated, and proposed filters that rely on potentially unstable or unmaintained packages will not be able to be implemented
```